pkg/logging/logging.go

```diff
@@ -130,6 +130,7 @@ func LogStructure(argName string, argStruct interface{}, log *zerolog.Logger, fo
 	if err != nil {
 		log.Warn().Msg("Failed converting " + argName + " to json")
 	} else {
-		log.Debug().Bool(FORUSER, forUser).Bool(AUDIT, audit).Msg(argName + ": " + string(jsonStruct))
+		// Log the info making sure that the calling function is listed as the caller
+		log.Debug().CallerSkipFrame(3).Bool(FORUSER, forUser).Bool(AUDIT, audit).Msg(argName + ": " + string(jsonStruct))
```

Comment:
- Please fix the golint error.
- why do you skip 3 frames, it provides the wrong output, you should skip a single frame (the `LogStructure` function itself"

Reply:
P.S. as I remember golint doesn't require to create a constant for "1"

Reply:
@roytman Golint does require a constant.  Also, skipping only one or two frames didn't solve the problem.  Only once I put 3 was it resolved.

Reply:
@simanadler , i can show you that the skip should be a single frame, see rego_file_evaluator.go:54, do you see a log statement there ? 
And golint is configured to skip "1", see https://github.com/fybrik/fybrik/blob/451c4192920c132350d83b1678edbff3f34c90a7/.golangci.yml#L64

Reply:
@roytman fixed